components/Benefit/Benefit.js

```diff
@@ -0,0 +1,105 @@
+import styled from "styled-components";
+import freeshippingicon from "./img/freeshippingicon.svg";
+import paymentsecurityicon from "./img/100-percent-payment-secure.svg";
+import thirtydaysreturn from "./img/30-days-return.svg";
+import supporticon from "./img/support247icon.svg";
+const Benefit = () => {
+  return (
+    <BenefitContainer>
+      <BenefitItem>
+      <BenefitIconContainer>
+        <FreeShippingIcon />
+        </BenefitIconContainer>
+        <BenefitDiv>
+          <BenefitTitle>FREE SHIPPING</BenefitTitle>
+          <BenefitDescription>
+            Free shipping on all US order or order above $100
```

Comment:
can we also move these texts to some constant file. It would be a better practice IMO to keep all static text in a constant file and pick values from there.Consider Making all your text logic controllable from one place.